readthedocs/doc_builder/python_environments.py

```diff
@@ -251,7 +251,7 @@ def install_core_requirements(self):
             # If this flag is present,
             # we need to cap setuptools again.
             # See https://github.com/readthedocs/readthedocs.org/issues/8775
-            requirements.append('setuptools<58.3.0')
+            requirements.append(setuptools_version)
```

Comment:
The comment above could seem to also need some follow-up :)

Reply:
:+1: